src/main/java/org/apache/commons/imaging/formats/tiff/TiffRasterData.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.tiff;
+
+/**
+ * Provides a simple container for floating-point data. Some TIFF files are used
+ * to store floating-point data rather than images. This class is intended to
+ * support access to those TIFF files.
+ */
+public class TiffRasterData {
+
+
+    private final int width;
+    private final int height;
```

Comment:
Would it make sense call these two `columns` and `rows` instead? Asking because I noticed the other docs appear to treat them as such. Happy with either option :+1:

Reply:
I elected to say with width and height.  I myself think in terms of number of rows and columns,  and would have preferred that choice. But I thought it would be better for the user if the variable names were consistent with the width and height arguments used elsewhere in Commons Imaging, as in the BufferedImage constructor, etc.

Reply:
All good, thanks for the explanation @gwlucastrig !